operator/src/main/resources/scripts/stop-server.py

```diff
@@ -2,23 +2,200 @@
 # Licensed under the Universal Permissive License v 1.0 as shown at
 # http://oss.oracle.com/licenses/upl.
 
+# This the the preStop hook script (stopServer.sh) that is specified by the domain resource.
```

Comment:
'specified by domain resource' -> 'specified by the WL operator for WL pod resources.'

I don't think there's a domain resource configurable for the hook.  The hook and the pod resource itself are setup by the operator.

Reply:
Yes you are correct as far as I know.  I can fix the wording.

Reply:
Marking resolved as the change was made.